pmd-java/etc/grammar/Java.jjt

```diff
@@ -2282,47 +2282,32 @@ void RSIGNEDSHIFT() #void:
 
 /* Annotation syntax follows. */
 
-void Annotation() #void:
+void Annotation():
 {}
 {
- LOOKAHEAD( "@" VoidName() "(" ( <IDENTIFIER> "=" | ")" ))
- NormalAnnotation()
-|
- LOOKAHEAD( "@" VoidName() "(" )
- SingleMemberAnnotation()
-|
- MarkerAnnotation()
-}
-
-void AnnotationBase(Node n) #void:
-{String name = null;}
-{
-   "@" name=VoidName() {n.setImage(name);}
+    "@" jjtThis.name=VoidName() [ AnnotationMemberList() ]
```

Comment:
This is something I tried to make the grammar cleaner. But maybe, using a setter would be better, especially if the field is shared (this could be the image field)